assets/src/edit-story/elements/text/output.js

```diff
@@ -58,14 +65,43 @@ function TextOutput({
     textAlign: textAlign ? textAlign : null,
     textDecoration,
     whiteSpace: 'pre-wrap',
-    ...generatePatternStyles(backgroundColor),
+    ...bgColor,
     ...generatePatternStyles(color, 'color'),
   };
 
+  const highlightStyle = {
+    ...fillStyle,
+    margin: 0,
+    backgroundColor: undefined,
```

Comment:
Is this correct? The `bgColor` if I read this right also sets `background-image` and might be some more. Perhaps we shouldn't set `...bgColor` at all for this mode and fork these two styles a bit higher?

Reply:
Fixed.